File: sistemas distribuídos/Thread1.java
class PrimeiraThread extends Thread {
    public void run() {
        for(int i=0; i < 50; i++){
            System.out.println("Thread1 executando!");
        }
    }
}

class SegundaThread extends Thread {
    public void run() {
        for(int i=0; i<50; i++){
            System.out.println("Thread2 em acao!");
        }
    }
}

public class Thread1 {
    public static void main(String[] args) {
        PrimeiraThread t1 = new PrimeiraThread();
        SegundaThread t2 = new SegundaThread();
        t1.start();
        t2.start();    
    }
}

File: sistemas distribuídos/0 - threads/thread1.py
import threading

def tarefa1():
    for i in range(50):
        print("Thread1 executando!")

def tarefa2():
    for i in range(50):
        print("Thread2 em acao!")

t1 = threading.Thread(target=tarefa1)
t2 = threading.Thread(target=tarefa2)
t1.start()
t2.start()